bc_obps/registration/api/user_operator.py

```diff
@@ -41,6 +42,12 @@
 
 
 ##### GET #####
+@router.get("/user-operator-status-from-user", response={200: UserOperatorStatus, codes_4xx: Message})
```

Comment:
We could probably refactor and reuse an existing endpoint, but since this is a temporary solution, I create a new one for ease of cleanup later.